python/pyiceberg/table/__init__.py

```diff
@@ -128,6 +144,25 @@ def _append_updates(self, *new_updates: TableUpdate) -> Transaction:
         self._updates = self._updates + new_updates
         return self
 
+    def _append_requirements(self, *new_requirements: TableRequirement) -> Transaction:
+        """Appends requirements to the set of staged requirements.
+
+        Args:
+            *new_requirements: Any new requirements.
+
+        Raises:
+            ValueError: When the type of requirement is not unique.
+
+        Returns:
+            Transaction object with the new requirements appended.
+        """
+        for requirement in new_requirements:
+            type_new_requirement = type(requirement)
+            if any(type(update) == type_new_requirement for update in self._updates):
```

Comment:
Should this be `_updates` or `_requirements`?

Reply:
auch, copy-paste. Thanks!